lib/DataStore.js

```diff
@@ -51,6 +53,18 @@ function DataStore(userStore, roomStore) {
         unique: true,
         sparse: false
     }, errLog("user id"));
+
+    this._privateKey = null;
+
+    if (pkeyPath) {
+        try {
+            this._privateKey = fs.readFileSync(pkeyPath, "utf8").toString();
+            log.info(`Private key loaded from ${pkeyPath} - IRC password encryption enabled.`);
```

Comment:
We should probably assert that the key is usable first so we can fail fast if `_privateKey` isn't a PEM file. Try encrypting/decrypting a test string in this constructor.